panel.py

```diff
@@ -39,9 +39,13 @@ def create_panel(self):
             self.tree.insert('','0','overall mood',text='Overall Mood') 
 
             for i in range(len(self.activities)-1,-1,-1):
-                self.tree.insert('','0','activity'+str(i),text=self.activities[i]) # This inserts the activites starting at the end of the list going to beggining because an activity is displayed,
+                self.tree.insert('','0','activity'+str(i),text=self.activities[i][0]) # This inserts the activites starting at the end of the list going to beggining because an activity is displayed,
```

Comment:
what was the problem before the [0] ?

Reply:
basically i added the [0] because i made the activities list into a 2d array and had it so it had the checked/unchecked value next to the activitiy

Reply:
yeah now I get it !! Thanks !!